site/docs/master/install-overview.md

```diff
@@ -26,6 +26,8 @@ velero install \
 
 When using node-based IAM policies, `--secret-file` is not required, but `--no-secret` is required for confirmation.
 
+This command will configure a `BackupStorageLocation` and `VolumeSnapshotLocation` with the `provider` fields set to the value of the `--provider` flag.  The `--provider` value must match the name of the relevant plugin.
```

Comment:
In the AWS example, `velero.io/aws` is the provider and the image name is `gcr.io/heptio-images/velero-aws-plugin`. What does it mean here for the provider to match the plugin name?

Reply:
The `provider` must match the first argument passed in at https://github.com/heptio/velero/blob/master/pkg/cmd/server/plugin/plugin.go#L41.

afaik this is only exposed to the user via the plugin's documentation. It has no bearing on the image name, which I think isn't great.

Reply:
I think this provider/plugin name and image name disconnect is something we should think about more deeply, though I don't know that we can get it in for v1.2. How can we make this more easily discoverable at the command line? Is that possible?

Reply:
Yeah this is going to be pretty confusing, I'll need to think about this more. Image name doesn't make sense as this will be different if you need to mirror the image in different registries. What if the plugin name had to be provided to `velero plugin add <name> <image>`, and this is what's used to register the plugin, instead of the plugin registering itself? I guess this would be a backwards incompatible change, though.

Something we can think about for plugins v2, but for v1 we might just need to rely on documentation.

Reply:
>Image name doesn't make sense as this will be different if you need to mirror the image in different registries.

I'm not sure I understand this statement. `velero plugin add <IMAGE NAME>` basically inserts the container image address for an init container. How does that interact with mirroring in different registries? I would assume it's just the name of the registry your cluster is using at the time.